assets/jwt-ptf-rotation.yaml

```diff
@@ -19,14 +19,13 @@ spec:
         conjur.org/secrets-destination: file
         conjur.org/jwt-token-path: /var/run/secrets/tokens/jwt
         conjur.org/conjur-secrets.test-app: |
-          - test-secrets-provider-p2f-jwt-app-db/url
-          - test-secrets-provider-p2f-jwt-app-db/username
-          - test-secrets-provider-p2f-jwt-app-db/password
+          - admin-username: <path to secret in Conjur Cloud>
+          - admin-password: <path to secret in Conjur Cloud>
```

Comment:
Why are there references here to Conjur Cloud?

Reply:
This was tested in a Conjur Cloud environment.